compiler-core/src/format.rs

```diff
@@ -1734,13 +1734,13 @@ impl<'comments> Formatter<'comments> {
         G: Fn(&mut Self, &'a A) -> Document<'a>,
     {
         match figure_formatting(arg) {
-            CallArgFormatting::Unlabelled(value) => format_value(self, value),
-            CallArgFormatting::Punned(label) => {
+            CallArgFormatting::WithoutLabel(value) => format_value(self, value),
```

Comment:
I think this is worse 😅  Can we stick with the original naming please for the other two and have something that fits for the new variant.

ShorthandLabelled ?

Reply:
Ok!